Sources/RunnerLib/GetDangerJSPath.swift

```diff
@@ -3,7 +3,7 @@ import Logger
 import ShellOut
 
 public func getDangerCommandPath(logger: Logger, args: [String] = CommandLine.arguments, shellOutExecutor: ShellOutExecuting = ShellOutExecutor()) throws -> String {
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/RunnerLib/GetDangerJSPath.swift;
  Line: 5;
-->

- :warning: Line should be 120 characters or less: currently 165 characters (`line_length`)